src/unstable-temp/DownloadImageModal/DownloadImageModal.tsx

```diff
@@ -279,11 +256,19 @@ export const UnstableTempDownloadImageModal = ({
 										releaseId={releaseId}
 										downloadUrl={downloadUrl}
 										rawVersion={rawVersion}
+										developmentMode={developmentMode}
 										modalActions={modalActions}
 										authToken={authToken}
 										{...(downloadConfig && {
-											downloadConfig: (model) =>
-												downloadConfig(deviceType, rawVersion, model),
+											downloadConfig: ({
+												appId,
+												releaseId,
+												deviceType: _deviceTypeSlug,
+												version,
+												...model
+											}) =>
+												// TODO: Change this to be passing the whole model
+												downloadConfig(deviceType, version, model),
```

Comment:
Just out of curiosity, why are we removing some properties from the model before passing it ?

Reply:
It felt weird to pass those model props as both separate arguments & as part of the model.
As in, which of the two should I be using?
Hence I added a TODO to do a major to change this after we release the feature (since preserving compatibility for now helps a lot while testing for regressions by just switching the rendition branch).